src/dex-ui/pages/dao/DashboardHeader.tsx

```diff
@@ -74,6 +87,9 @@ export function DashboardHeader(props: DashboardHeaderProps) {
           <Flex height="40px" alignItems="center">
             <Breadcrumb to="/daos" label="Back to DAOs" />
           </Flex>
+          <Flex height="40px" alignItems="center">
+            {showMintNFTButton && token && <MintNFTModal token={token} handleMintNFT={handleMintNFT} />}
+          </Flex>
           <Flex direction="column" gap="0.6rem">
```

Comment:
if `showMintNFTButton` is `false` then we are adding extra `Flex` which will be extra gap between elements in UI, 
we could do this:

`          {
            showMintNFTButton ? (
              <Flex height="40px" alignItems="center">
                {showMintNFTButton && token && <MintNFTModal token={token} handleMintNFT={handleMintNFT} />}
              </Flex>
            ) : undefined
          }
`

Reply:
You are right. made the suggested changes.